eZ/Bundle/EzPublishCoreBundle/Session/Handler/NativeSessionHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * File containing the SymfonyNativeSessionHandler class.
```

Comment:
Nitpick: Should be `NativeSessionHandler` instead of `SymfonyNativeSessionHandler`. I'm not sure if this line is necessary ?

Reply:
I am not sure either.... I just saw such comments all over the place, so I added one too

Reply:
can be skipped

Reply:
but please add class phpdoc explaining what it is for

Reply:
Added class phpdoc, but kept line above copyright notice.  Such line seems to be present in every class in core, so I didn't want to introduce such inconsistency